### ISSUE TITLE
Addon Crash

### ISSUE THREAD

**Issue by Schrotti1290** [SAIC-iSmart-API/documentation#20]
Today the Addon starts to crash on its own.
![Fehlermeldung](https://github.com/SAIC-iSmart-API/documentation/assets/146772142/0c45cb37-13a3-45f1-a99b-54ef52d63dee)

**Comment by BerrisNO** [comment]
Same here

**Comment by lbourdel** [comment]
When I debug, message value is 'Blocked by Sentinel (flow limiting)'

Return value is not HEX, so fromhex() fct crashs @ @!!

**Comment by GamEased** [comment]
Same here. Any ideas?

**Comment by Schrotti1290** [comment]
After login into the MG App on my Phone and then restarting the API Addon it works for 6 hours now.

**Comment by GamEased** [comment]
Yes this is apparently the solution as I opened an issue in another repo.

**Comment by svenmayersm** [comment]
same problem since 2 weeks. anything in the pipeline to solve the problem?
